tests/test_helpers.py

```diff
@@ -0,0 +1,33 @@
+from django.contrib.contenttypes.models import ContentType
+
+from djangocms_versioning.constants import DRAFT, PUBLISHED
+from djangocms_versioning.models import Version
+from filer.models import File, Folder
+
+from .base import BaseFilerVersioningTestCase
+from djangocms_versioning_filer import helpers
+
+
+class HelperTests(BaseFilerVersioningTestCase):
```

Comment:
Do you definitely need everything that the `setUp` method in `BaseFilerVersioningTestCase` does? Cause I glanced at it and it does a looooot of things and most of those things involve creating things in the db. This means that with every test in this test case you're probably creating around 10 records in the db in addition to data you're creating in the test itself. Overusing such classes is one of the ways people end up with excruciatingly slow test suites that stop doing their job properly, so I would really caution against that.